hydra-access-controls/lib/hydra/policy_aware_access_controls_enforcement.rb

```diff
@@ -1,36 +1,36 @@
 # Repeats access controls evaluation methods, but checks against a governing "Policy" object (or "Collection" object) that provides inherited access controls.
 module Hydra::PolicyAwareAccessControlsEnforcement
 
-  # Extends Hydra::AccessControlsEnforcement.apply_gated_discovery to reflect policy-provided access
-  # appends the result of policy_clauses into the :fq
-  # @param solr_parameters the current solr parameters
-  # @param user_parameters the current user-subitted parameters
+  # Extends Hydra::AccessControlsEnforcement.apply_gated_discovery to reflect policy-provided access.
+  # Appends the result of policy_clauses into the :fq
+  # @param [Hash] solr_parameters the current solr parameters, to be modified herein!
   def apply_gated_discovery(solr_parameters)
     super
     logger.debug("POLICY-aware Solr parameters: #{ solr_parameters.inspect }")
   end
 
-  # returns solr query for finding all objects whose policies grant discover access to current_user
+  # @return [String,void] solr query for finding all objects whose policies grant discover access to current_user
```

Comment:
Could use `nil`.  I was going from https://github.com/lsegal/yard/blob/master/docs/GettingStarted.md 
`Another meta-type is "void", which stands for "no meaningful value" and is used for return values. These meta-types are by convention only, but are recommended.`